lib/coverband/utils/source_file.rb

```diff
@@ -171,7 +179,7 @@ def lines_strength
       end
 
       def relevant_lines
-        lines.size - never_lines.size - skipped_lines.size
+        @runtime_relavant_lines || (lines.size - never_lines.size - skipped_lines.size)
```

Comment:
I apologize as I am not as familiar with this area of the codebase.. would it be possible to make a `runtime_relevant_lines` method and instead invoke this method from the erb when needed?

Reply:
I don't think so because what we are actually calling is formatted coverage percentage, which in turn calls a number of other methods, this is the root of what changes the values of the calculations, but isn't the actual method... basically, the other approach requires having runtime_relavent_lines, runtime_relavent_coverage, and formatted_runtime_coverage and perhaps more

Reply:
Ok makes sense. Do you think that having all of those extra methods might be worth it at some point? I think it might make the code more straight forward.

Reply:
I think it would make things much more complicated because if you need to change any of the calculations or have a bug in one you now need to fix it in the original implementation and the duplicated methods. They would all be exactly the same except for using this base data.

I generally believe every line of code has a maintenance cost and opportunity to add bugs so minimizing lines of code by using slightly more complicated abstractions feels better to me.

Reply:
Makes sense.